gatsby_fp/src/components/fieldComponents/univService.js

```diff
@@ -0,0 +1,77 @@
+import React from "react"
+
+export default function UnivService ({univServiceList}){
+  return(
+    
+    <>
+    <div className="sectiontitle">Institutional Service</div>
+        <div>
+          <ul>
+          {univServiceList.map((element, index) => (
+            // <li key={index} dangerouslySetInnerHTML ={{__html: 
+            // `${element.scope}<br>${element.member_type} of ${element.org}
+            // (${element.dtm_start} ${element.dty_start} - ${element.dtm_end} 
+            // ${element.dty_end})<br> Compenstated:  ${element.compensation}`}}>
+            // </li>
+            <li key={index} dangerouslySetInnerHTML ={{__html:` 
```

Comment:
We should avoid using indices as keys. (https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318)